python/api/osrd_infra/views/timetable.py

```diff
@@ -9,21 +9,13 @@
 
 class TimetableView(
     mixins.CreateModelMixin,
-    mixins.RetrieveModelMixin,
     mixins.UpdateModelMixin,
     mixins.DestroyModelMixin,
-    mixins.ListModelMixin,
     GenericViewSet,
 ):
     serializer_class = TimetableSerializer
     pagination_class = CustomPageNumberPagination
-
-    def get_queryset(self):
-        queryset = Timetable.objects.order_by("-pk")
-        infra = self.request.query_params.get("infra")
-        if infra is not None:
-            queryset = queryset.filter(infra__pk=infra)
-        return queryset
+    queryset = Timetable.objects.order_by("-pk")
 
     def retrieve(self, request, pk):
```

Comment:
if the retrieve mixing is to be removed, the method has to go too

Reply:
i have kept retrieve and removed create mixins :)